README.md

```diff
@@ -53,33 +53,33 @@ app
 │   │   ├── effects.js => Async actions creators, always return a promise
 │   │   ├── record.js => The object representing a single comment
 │   │   ├── reducer.js => Reducer + actions creators, always return an action object
-│   │   └── selectors.js => re-select selectors to select data in state
+│   │   └── selectors.js => Re-select selectors to select data in state
 │   └── ...
-├── static => static assets, directly copied to the public directory
-├── styles => stylesheets in .sass format, all included from application.sass
+├── static => Static assets, directly copied to the public directory
+├── styles => Stylesheets in .sass format, all included from application.sass
 └── router.jsx => Application router and main entry point
 ```
 
 #### Styling
 
 Styling is based on [Bulma](https://bulma.io/documentation/) and was initially
-customized with SASS. However we now include [styled-components](https://github.com/emotion-js/emotion), [styled-system](https://github.com/jxnblk/styled-system) and [@rebass/grid](https://github.com/rebassjs/grid)
-. This has become the prefered way to build new components.
+customized with SASS. However we now include [styled-components](https://github.com/emotion-js/emotion), [styled-system](https://github.com/jxnblk/styled-system) and [@rebass/grid](https://github.com/rebassjs/grid).
+This has become the prefered way to build new components.
 
 #### Icons
 
-We bundle a custom font icon built with icomoon. You can find scripts and config
+We bundle a custom font icon built with [IcoMoon](https://icomoon.io/). You can find scripts and config
 for this in `dev/` but the preferred (and easiest) way to add new icons today
 is to use `https://styled-icons.js.org/`.
 
 ## Main Libraries / Frameworks
 
-- ES6 with Babel
-- ReactJS: vue layer
-- Redux: state management
-- ReactRouter: routing
-- phoenix: interaction with phoenix socket
-- Bulma: base styles
+- [ES6](http://es6-features.org) with [Babel](https://babeljs.io/)
+- [React](https://reactjs.org/): [Vue.js](https://vuejs.org/) layer
```

Comment:
This was a typo in the original sentence, we don't use Vue. Sentence was intended to be `ReactJS: view layer`